test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,18 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void elementsAreNull() {
+        //no arguments
+        assertFalse(Utils.isAnyNull());
+
+        //one null object
+        assertTrue(Utils.isAnyNull( (Object) null));
+
+        //null object within a list of non-null objects
+        assertTrue(Utils.isAnyNull((Object) null, 0, "string"));
+        assertTrue(Utils.isAnyNull(0, "string", (Object) null));
+        assertTrue(Utils.isAnyNull( 0, (Object) null, "string"));
```

Comment:
This is definitely okay, but we can go on to split this further if we want to:

Recall that in class, we went through a possible way to structure the tests: 
1. Test for empty list
2. Test for any non-empty list
3. Test for non-empty list with null at the start
4. Test for non-empty list with nulls in the middle
5. Test for non-empty list with one null at the end
6. Test to confirm that nulls inside the list are not considered:
```
List<Object> nullList = Arrays.asList((Object) null);
assertFalse(Utils.isAnyNull(nullList));
``` 

Your tests here are similar to these, but you can see they are not as specifically divided as this one :)